modules/custom/az_event/templates/field--node--field-az-event-date--az-event.html.twig

```diff
@@ -0,0 +1,96 @@
+{#
+/**
+ * @file
+ * Theme override for a field.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - multiple: TRUE if a field can contain multiple items.
+ * - items: List of all the field items. Each item contains:
+ *   - attributes: List of HTML attributes for each item.
+ *   - content: The field item's content.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ *
+ * @see template_preprocess_field()
+ */
+#}
+{%
+  set classes = [
+    'field',
+    'field--name-' ~ field_name|clean_class,
+    'field--type-' ~ field_type|clean_class,
+    'field--label-' ~ label_display,
+  ]
+%}
+{%
+  set title_classes = [
+    'field__label',
+    'h3',
+    label_display == 'visually_hidden' ? 'visually-hidden',
+  ]
+%}
+
+{% if label_hidden %}
+  {% if multiple %}
+    <div{{ attributes.addClass(classes, 'field__items') }}>
+      {% if element['#view_mode'] == 'full' and items|length > 1 %}
+        {% set accordion_item_id = element['#object'].id %}
+        {% for item in items %}
+          {% if item['content']['#nextInstance'] == true %}
+            <div class="mb-3">
+              <div{{ item.attributes.addClass('field__item') }}>Next event: {{ item.content }}</div>
+            </div>
+          {% endif %}
+        {% endfor %}
+        <div class='accordion h-100'>
+          <div class="card">
+            <div class="card-header">
+              <h3 class="my-0">
+                <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#event-{{ accordion_item_id }}" aria-expanded="false" aria-controls="event-{{ accordion_item_id }}">
+                  View all event instances
```

Comment:
Can this button have less padding so it's not so prominent?

Reply:
@danahertzberg My goal was to use the standard AZ Accordion styling for the list of event instances, but maybe we could modify AZ Accordion to allow adjustment of that padding? Could this be a follow-up issue?

Reply:
Here is an alternative using button dropdowns: https://digital.arizona.edu/arizona-bootstrap/docs/2.0/components/dropdowns/#text

Reply:
Per AZ Digital discussion:

- Dana is OK with adjusting the accordion styling as a follow-up.
- We decided against the button dropdown since the list of recurring events can be very, very long (see the example from #4141).